apps/github-cascading-app/src/cascading/cascading.ts

```diff
@@ -364,7 +364,14 @@ ${MESSAGE_NEW_UPDATE} ${baseBranch}
         return;
       }
     } else {
-      await this.createBranch(cascadingBranch, currentBranch.branch);
+      try {
+        await this.createBranch(cascadingBranch, targetBranch.branch);
+        await this.merge(currentBranch.branch, cascadingBranch);
```

Comment:
![image](https://github.com/AmadeusITGroup/otter/assets/122815580/b8274ece-a159-4bd3-be6a-4066999fc266)
In case of a 403, delete / create won't make a difference, same for 404 even if it should not happen. In case of 422 it looks just like a retry.We are left with a 409, shouldn't we keep the merge conflict so it's shown in the branch ? Do we want to delete/create in this case ? If I remember correctly we had an issue while create the PR with the branch after the create ref, that the build for the PR wasn't considering the pipeline changes from the target branch.

Reply:
Actually the try/catch is mainly for the 409. The problem is that if this merge fail we will do a PR without any diff, not a config.
The way to be able to create a conflicting PR (I put in place in this PR) is to create the PR from the previous release branch (without merge then). It is creating a conflicting PR that can not run the CI (because conflicting).
The drawback it that according the way the conflict it fix it may run the CI from the previous release branch indeed.
If you think about a better way I will take it :)

Reply:
After discussion with the limits running it on Azure functions, I agree with the changes